eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/AbstractHTTPServer.java

```diff
@@ -703,14 +720,15 @@ protected void initChannel(final SocketChannel channel) {
             if (sslContext != null && useTLS) {
                 final SSLEngine sslEngine = sslContext.createSSLEngine();
                 sslEngine.setUseClientMode(false);
-                pipeline.addFirst("ssl", new SslHandler(sslEngine));
+                pipeline.addFirst(getWorkerGroup(), "ssl", new SslHandler(sslEngine));
```

Comment:
- The bossGroup is used to listen and poll Accept requests

- The ioGroup is mainly used to listen for READ and WRITE events. It handles channels registered by the Boss Group.

- The workGroup is used to execute  ChannelHandlers

All ChannelHandlers can be executed using a single thread pool without the need for separate thread pools, but in this case, the business processing is handled using a separate thread pool.

Reply:
Thanks, I mean bossGroup and ioGroup pipeline whether need to add 'SslHandler'

Reply:
> Thanks, I mean bossGroup and ioGroup pipelines need to add 'SslHandler'? You need to check the stage of SSL work, because when the old logic, you don't need to consider this issue.

@mytang0  The bossGroup and ioGroup pipelines do not need to add 'SslHandler' because data processing is performed in the workerGroup, and the SslHandler is only used when data is involved.